-------------------------------------------------------------------------------
Test set: redis.clients.jedis.tests.JedisClusterTest
-------------------------------------------------------------------------------
Tests run: 53, Failures: 0, Errors: 53, Skipped: 0, Time elapsed: 0.096 sec <<< FAILURE! - in redis.clients.jedis.tests.JedisClusterTest
testClusterCountKeysInSlot(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testClusterCountKeysInSlot(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.003 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testRedisClusterMaxRedirections(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testRedisClusterMaxRedirections(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testClusterKeySlot(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testClusterKeySlot(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testSetClientName(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testSetClientName(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testRedisHashtag(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testRedisHashtag(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testAskResponse(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testAskResponse(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testIfPoolConfigAppliesToClusterPools(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testIfPoolConfigAppliesToClusterPools(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testReadonly(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testReadonly(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testDiscoverNodesAutomatically(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testDiscoverNodesAutomatically(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testMigrateToNewNode(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testMigrateToNewNode(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testMovedExceptionParameters(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testMovedExceptionParameters(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testThrowMovedException(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testThrowMovedException(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testCloseable(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testCloseable(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testLocalhostNodeNotAddedWhen127Present(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.003 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testLocalhostNodeNotAddedWhen127Present(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.004 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testClusterFlushSlots(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testClusterFlushSlots(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testClusterForgetNode(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.003 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testClusterForgetNode(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.004 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testInvalidStartNodeNotAdded(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testInvalidStartNodeNotAdded(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testCalculateConnectionPerSlot(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.009 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testCalculateConnectionPerSlot(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.009 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testMigrate(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.007 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testMigrate(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.008 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testReturnConnectionOnJedisConnectionException(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.003 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testReturnConnectionOnJedisConnectionException(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.004 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testThrowAskException(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testThrowAskException(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.003 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testStableSlotWhenMigratingNodeOrImportingNodeIsNotSpecified(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testStableSlotWhenMigratingNodeOrImportingNodeIsNotSpecified(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.003 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testReturnConnectionOnRedirection(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.004 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testReturnConnectionOnRedirection(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.004 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testJedisClusterTimeout(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testJedisClusterTimeout(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.003 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testRecalculateSlotsWhenMoved(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testRecalculateSlotsWhenMoved(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.003 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

testJedisClusterRunsWithMultithreaded(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.003 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:71)

testJedisClusterRunsWithMultithreaded(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.004 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:131)

redis.clients.jedis.tests.JedisClusterTest  Time elapsed: 0.005 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:119)

